examples/react-app/src/components/KnowledgeGraph.scss

```diff
@@ -0,0 +1,6 @@
+.KnowledgeGraph {
+  min-height: 680px;
```

Comment:
Suggest converting these sizes to use `rem` so that they're properly responsive. `px` tends to be monitor-specific.

Reply:
Could also implement `vh` & `vw` if you want the size based on the browser window.

Reply:
Two things I would like to clarify, This change has nothing to do with the PR purpose just while testing my change using the react example I realized the KG component in the example is styles with a class that didn't exist so I added the class which has the same CSS as the video player and the Scene viewer both use px.
The other thing is I can't change this in isolation if the rest of the components are using PX this will mess up the whole app if the user change the default browser size from 16 to any value the knowledge Graph will be the only component that will adjust coz its using rem.

I don't disagree with rem but it should be part of a comprehensive evaluation and migration (or using both rem & px) to all the components in the App so we don't end up with inconsistent behavior. This argument actually apply to the below comments as well.